cmd/code/push.go

```diff
@@ -46,7 +46,7 @@ func NewPushCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			if watch && code.IsCodeParsing(currentVersion) {
+			if !notWait && code.IsCodeParsing(currentVersion) {
```

Comment:
Double negative validations are always a bad idea.
I would define a param above:
const watch = !notWait
and give it a comment even to explain why